packages/db/db-client-types.ts

```diff
@@ -192,15 +195,15 @@ export type UploadItem = {
 }
 
 export type UploadItemOutput = {
-  _id: string
+  _id?: string
   type: definitions['upload']['type']
   name?: definitions['upload']['name']
   created: definitions['upload']['inserted_at']
   updated: definitions['upload']['updated_at']
   cid: definitions['content']['cid']
   dagSize?: definitions['content']['dag_size']
   pins: Array<PinItemOutput>,
-  deals: Array<Deal>
+  deals?: Array<Deal>
```

Comment:
Same as above 🙏

Reply:
Ok reinstated as above, deals required by `listUploads` and `getUpload`